<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGTLiftPlatModule</class>
 <widget class="QWidget" name="QGTLiftPlatModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGTLiftPlatModule</string>
  </property>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>温度 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>输出脉宽 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Motor_voltage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Motor_current">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_En_Status_explain">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>禁止</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Modbus_Status_explain">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>禁止</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Speed_start">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Motor_temperature">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_LiftPlat_Stop">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>220</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_Acceleration">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(℃)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Motor_Speed_actual">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>当前转速 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_En_Status">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(r/min)/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Pu_AllStep">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>电流 :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Motor_TargetSpeed">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(A)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>230</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Modbus_Status">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>En使能 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Plat_Height">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>460</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Target_Speed">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(r/min)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>460</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>当前高度 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>电机起始转速 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_LiftPlat_MoveUp">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>50</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(mm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>电机转速 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(r/min)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Motor_PWM">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>电机加速度 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Pu_Step">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>目标转速 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGTToolButton" name="toolButton_LiftPlat_MoveDown">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>390</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Pu步数 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>modbus使能 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>电压 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>390</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>(r/min)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Pu总步数 :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGTToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">Commen/qgttoolbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
